src/stories/smart-components/adres-crab.stories.ts

```diff
@@ -13,10 +14,25 @@ const meta: Meta<typeof AdresCrab> = {
       return { args };
     },
     template: `
+      <div>
       <Suspense>
-        <AdresCrab v-bind="args" />
+        <AdresCrab v-bind="args" @update:adres="onUpdateAdres" />
       </Suspense>
+      <h3>Adres:</h3>
+      <pre>{{ eventOutput }}</pre>
```

Comment:
Niet echt een groot probleem, maar in het documentatie code block wordt nu ook wel altijd een `h3` en `pre` tag mee gerendered
![image](https://github.com/OnroerendErfgoed/vue_component_library/assets/69460082/790407e1-1878-47ca-8638-5a06fd11f187)

Reply:
ah lap. valt deze te vermijden?

Reply:
Niet direct dat ik weet, aangezien die `autodocs` gewoon de template gaat uitlezen

Reply:
op zich kan deze niet zo veel kwaad idd, het is altijd wel nuttig om die output te zien. in de toekomst overwegen om die output enkel in de console log te  tonen